src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -84,22 +92,22 @@
     /** Period literal. */
     private static final String PERIOD = ".";
 
-    /**
-     * Stores allowed values in document for inherit doc literal.
-     */
-    private static final Set<Integer> SKIP_TOKENS = new HashSet<>(
-        Arrays.asList(JavadocTokenTypes.NEWLINE,
-                       JavadocTokenTypes.LEADING_ASTERISK,
-                       JavadocTokenTypes.EOF)
+    /** Inherit doc literal. */
+    private static final String INHERIT_DOC = "{@inheritDoc}";
```

Comment:
@rnveach can you please tell me how we can match against token?
i think we need to use absolute string to match. Let me know if i am wrong.